hudi-client/hudi-spark-client/pom.xml

```diff
@@ -102,14 +102,6 @@
       <version>${hbase.version}</version>
       <scope>test</scope>
       <exclusions>
-        <exclusion>
-          <groupId>org.codehaus.jackson</groupId>
-          <artifactId>jackson-mapper-asl</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.codehaus.jackson</groupId>
-          <artifactId>jackson-core-asl</artifactId>
-        </exclusion>
```

Comment:
@xushiyan From my personal point of view, we have completely removed all dependencies of  `org.codehaus.jackson`  from the parent project's pom. Do these excludes need to be retained? We shouldn't need to exclude anything since there are no dependencies.

Reply:
I understand what you mean, I will make changes as it may depend on other projects containing `org.codehaus.jackson`.